ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-pathfinder.cfg

```diff
@@ -118,6 +118,7 @@ router path-selection
    tcp mss ceiling ipv4 ingress
    !
    path-group Equinix id 103
+      keepalive interval 100 milliseconds failure-threshold 12 intervals
```

Comment:
do we need the keepalive line rendered if there is no local interface?

Reply:
No, we do not. Removed.